File: app/src/main/java/com/future/awaker/db/dao/CommentListDao.java
package com.future.awaker.db.dao;


import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Insert;
import android.arch.persistence.room.OnConflictStrategy;
import android.arch.persistence.room.Query;

import com.future.awaker.db.entity.CommentEntity;

import io.reactivex.Flowable;

@Dao
public interface CommentListDao {

    @Query("SELECT * FROM comment_entity WHERE id = :id")
    Flowable<CommentEntity> loadCommentEntity(String id);

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    void insertCommentEntity(CommentEntity commentEntity);
}

File: article/src/main/java/com/ruzhan/awaker/article/db/dao/CommentListDao.kt
package com.ruzhan.awaker.article.db.dao


import android.arch.persistence.room.Dao
import android.arch.persistence.room.Insert
import android.arch.persistence.room.OnConflictStrategy
import android.arch.persistence.room.Query

import com.ruzhan.awaker.article.db.entity.CommentEntity

import io.reactivex.Flowable

@Dao
interface CommentListDao {

    @Query("SELECT * FROM comment_entity WHERE id = :id")
    fun loadCommentEntity(id: String): Flowable<CommentEntity>

    @Insert(onConflict = OnConflictStrategy.REPLACE)
    fun insertCommentEntity(commentEntity: CommentEntity)
}
